cluster/devel/kubevirt-ssp-operator.yaml

```diff
@@ -177,7 +181,7 @@ spec:
         type: Recreate
       containers:
         - name: kubevirt-ssp-operator
-          image: quay.io/fromani/kubevirt-ssp-operator-container:latest
+          image: quay.io/oyahud/kubevirt-ssp-operator-container@sha256:5b415c7cd34f9e2b6e3fc2e2243d2df3681f160914654961f9385d49afb34c64
```

Comment:
Do we need this change?

Reply:
This file should  have been removed, fixed